lottery.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+numbers = []
+winning_num = []
+6.times do |i|
+  print "[#{i + 1}] Enter a number between 1 and 20: "
+  numbers << gets.to_i
+end
+
+puts "The winning numbers are:"
+6.times do
+  loop do
+    winning_numbers = rand(1..20)
+    unless winning_num.include?(winning_numbers)
+      winning_num << winning_numbers
+      break
+    end
+  end
+end
+
+puts "[1] #{winning_num}"
+
+puts "You have guessed #{(winning_num & numbers).length} numbers"
```

Comment:
Can you also display which winning numbers the user was able to guess?
Like so

__The winning numbers you were able to guess are [23. 12, 1].__